contracts/external/interfaces/gmx/IGmxEventEmitter.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: BUSL-1.1
+
+pragma solidity ^0.8.9;
+
+import { EventUtils } from "./GmxEventUtils.sol";
+
+interface IGmxEventEmitter {
+    event EventLog(
```

Comment:
Add new line between first thing in contracts/interfaces and the contract/interface name